lib/msal-electron-proof-of-concept/src/AppConfig/ClientApplication/ClientApplicationBase.ts

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-import { ApplicationConfiguration } from './ApplicationConfiguration';
-import { AuthOptions } from './AuthOptions';
+import { ApplicationConfiguration } from '../ApplicationConfiguration';
+import { AuthOptions } from '../AuthOptions';
 
 /**
  * ClientApplicationBase abstract class
```

Comment:
I don't you need "Base" in the name as your inherited class is 'PublicClientApplication'. You can rename this 'ClientApplication' :)